connectors-and-apis.md

```diff
@@ -74,7 +74,6 @@ The third-party APIs are not developed by Oracle. The following table lists the
 | Ruby           | MySQL/Ruby                               | `libmysqlclient`                 | Uses `libmysqlclient`. See [Section 27.12.1, “The MySQL/Ruby API”](https://dev.mysql.com/doc/refman/5.7/en/apis-ruby-mysqlruby.html) |
 |                | Ruby/MySQL                               | Native Driver                    | See [Section 27.12.2, “The Ruby/MySQL API”](https://dev.mysql.com/doc/refman/5.7/en/apis-ruby-rubymysql.html) |
 | Scheme         | `Myscsh`                                 | `libmysqlclient`                 | See [`Myscsh`](https://github.com/aehrisch/myscsh) |
-| SPL            | `sql_mysql`                              | `libmysqlclient`                 | See [`sql_mysql` for SPL](http://www.clifford.at/spl/spldoc/sql_mysql.html) |
```

Comment:
This document is not displayed on docs.pingcap.com and needs rigorous review and update. I don't think it's necessary to delete this entire row.

Reply:
Only removing the link is the best option for now.

Reply:
I've opened https://github.com/pingcap/docs/issues/6848 to address the other issues with this document